include/ews/ews.hpp

```diff
@@ -11935,154 +13223,164 @@ namespace ews
     {
     public:
         //! Use this constructor if you want to delete a property from an item
-        explicit property(property_path path) : path_(std::move(path)), value_()
+        explicit property(property_path path) : value_()
         {
+            value_ = path.to_xml();
         }
```

Comment:
You should initialize a member in the initializer list and not in the constructor body.

Here and other locations.